src/controllers/2FAcontroller.js

```diff
@@ -0,0 +1,21 @@
+/* eslint-disable no-else-return */
```

Comment:
Put these contents in a user controller file

Reply:
> Put these contents in a user controller file

the whole content of the 2FA controller ? i thought the two factor controller has its own controller